src/backend/access/external/pxffilters.c

```diff
@@ -719,6 +1029,143 @@ const_to_str(Const *constval, StringInfo buf)
 
 
 /*
+ * list_const_to_str
+ *
+ */
+static void
+list_const_to_str(Const *constval, StringInfo buf)
+{
+	StringInfo	interm_buf;
+	Datum *dats;
+	ArrayType  *arr;
+	int len;
+
+	if (constval->constisnull)
+	{
+		elog(DEBUG1, "Null constant is not expected in this context.");
+		return;
+	}
+
+	if (constval->constbyval) {
```

Comment:
This check seems a bit unnecessary as we would never get this expression down to our code